force_wfmanager/tests/test_wfmanager.py

```diff
@@ -23,47 +24,57 @@ def setUp(self):
         super(TestWfManager, self).setUp()
         self.wfmanager = ProbeWfManager()
 
+    @contextlib.contextmanager
```

Comment:
Out of curiosity, is use of this library preferred when performing UI tests or just a work around for this particular case?

Reply:
I had originally created a "cleanup" method that would mirror `create_tasks`, in order to arrive at the `tearDown` stage "prepared". Since that cleanup had to be run for any test that uses `create_tasks`, and since a proper clean up was nice to have also if the test errored/failed, having it wrapped in a context manager helped.

Reply:
Cheers, makes sense - it looks neat, so I was wondering what other situations would it might be useful for.

Reply:
I'd say in every case where you have some cleanup tasks that are not completely general for the test class (which go in `tearDown`) and have to happen before `tearDown` (`addCleanup` happens after)